spec/develop-spec.js

```diff
@@ -28,7 +29,6 @@ describe("apm develop", function () {
 
   describe("when the package doesn't have a published repository url", () =>
     it("logs an error", function () {
-      const Develop = require("../lib/develop")
```

Comment:
I don't think spies are preserved between the tests. However, since this repository is using jasmineFocused, we need to check that. @UziTech Do you know if this is required?